packages/terra-paginator/docs/UPGRADEGUIDE.md

```diff
@@ -34,3 +34,37 @@
 * --terra-paginator-nav-link-icon-width
 * --terra-paginator-nav-link-icon-previous-margin-right
 * --terra-paginator-nav-link-icon-next-margin-left
+
+#### Added ~2.15.x
+// todo === finish editing ====
```

Comment:
You'll want to remove this comment if you're done editing.

Reply:
Removed. https://github.com/cerner/terra-core/pull/2485/commits/e52464898ce1177b2b3f14d5d247fa5778aca27b

Reply:
It looks like the todo comment is in [the file](https://github.com/cerner/terra-core/blob/b3e6b50505a5a5620628ccc1ee4c3285e0ad17d0/packages/terra-paginator/docs/UPGRADEGUIDE.md) still. The [commit linked](e524648) didn't show a removal of that comment.